README.md

```diff
@@ -1,3 +1,33 @@
 # fasjson-client
 
 A python client library for the FASJSON API
+
+This client uses the bravado library to build dynamic api methods based on open-api specs (version 2.0): https://github.com/Yelp/bravado
+
+## Usage
+
+```
+c = Client.from_url('http://myspecurl.com', base_url='http://fasjson.example.test/fasjson' principal='admin@EXAMPLE.TEST')
+
+output = c.api.v1.whoami().response().result
```

Comment:
Could we propose something more direct? Like `c.whoami()`, ideally.

Reply:
is just uses whatever is in the url path, so  `/v1/whoami` becomes `v1.whoami()`, to use `whoami()`directly we need to remove  `v1` path from the url.

Reply:
Maybe not removing from the server, but only hide it in the client? It feels like our users shouldn't have to know what boils down to an implementation detail. Same thing for the `.response().result` part. I'm all ears if you have a different opinion.

Reply:
I meant this is auto generated by bravado, the `/v1/whoami` thing is read from the api spec... I could do some method naming fu using "__getarr_" to retrieve/call methods within the `v1` object.. but it doesn't feel right, I don't jhave a problem with "v1", we could remove it from the spec since we only have a version of the api now.

About `response().result` that bravado again because response() returns a future so you can handle async code if you want to... I could try to use the __getattr__ method here again.

Reply:
github messed up with my comment, I wanted to write `__ getattr __` :P

Reply:
an alternative would be to have different specs per api version and add the `/v1` part in the basePath field.